controllers/errors.js

```diff
@@ -1,8 +1,25 @@
-exports.handle404s = (req, res, next) => {
-  res.status(404).send({ msg: "Cannot find what you wanted" });
+exports.handleCustomErrors = (req, res, next) => {
+  res
+    .status(404)
+    .send({ msg: "You tried to look for something and it wasn't found" });
+};
+
+exports.handle400s = (err, req, res, next) => {
+  if (err.status === 400) {
+    res.status(400).send({ msg: "Bad request" });
+  } else {
+    next(err);
+  }
 };
 
 exports.handle500s = (err, req, res, next) => {
-  console.log(err);
-  res.status(500).send({ msg: "Error in code somewhere" });
+  if (err.status === 500) {
```

Comment:
You can't test for 500's they are the final source of an issue if nothing is caught by your previous handlers. The if statement isn't needed and won't work in this way. Your handle500s should be at the bottom of errors.js and therefore doesn't need a next(err) either.